JustSaying/AwsTools/JustSayingConstants.cs

```diff
@@ -2,70 +2,70 @@ namespace JustSaying.AwsTools
 {
     public static class JustSayingConstants
     {
-        public const string ATTRIBUTE_REDRIVE_POLICY = "RedrivePolicy";
-        public const string ATTRIBUTE_ARN = "QueueArn";
-        public const string ATTRIBUTE_RETENTION_PERIOD = "MessageRetentionPeriod";
-        public const string ATTRIBUTE_VISIBILITY_TIMEOUT = "VisibilityTimeout";
-        public const string ATTRIBUTE_DELIVERY_DELAY = "DelaySeconds";
-        public const string ATTRIBUTE_POLICY = "Policy";
-        public const string ATTRIBUTE_ENCRYPTION_KEY_ID = "KmsMasterKeyId";
-        public const string ATTRIBUTE_ENCRYPTION_KEY_REUSE_PERIOD_SECOND_ID = "KmsDataKeyReusePeriodSeconds";
+        public const string AttributeRedrivePolicy = "RedrivePolicy";
+        public const string AttributeArn = "QueueArn";
+        public const string AttributeRetentionPeriod = "MessageRetentionPeriod";
+        public const string AttributeVisibilityTimeout = "VisibilityTimeout";
+        public const string AttributeDeliveryDelay = "DelaySeconds";
+        public const string AttributePolicy = "Policy";
+        public const string AttributeEncryptionKeyId = "KmsMasterKeyId";
+        public const string AttributeEncryptionKeyReusePeriodSecondId = "KmsDataKeyReusePeriodSeconds";
 
         /// <summary>
         /// Default visibility timeout for message in seconds
         /// </summary>
-        public static int DEFAULT_VISIBILITY_TIMEOUT = 30;
+        public static int DefaultVisibilityTimeout = 30;
```

Comment:
Again, separate to renaming, but I feel these should be properties, not fields, for the same reasons as above.

Reply:
Yep, the analysers complain that these are public mutable fields. It should not be at least one of those things. But in a separate PR.